core/src/main/java/nl/nn/adapterframework/jdbc/dbms/GenericDbmsSupport.java

```diff
@@ -269,102 +266,91 @@ public String getSchema(Connection conn) throws JdbcException {
 		return null;
 	}
 
-	protected boolean doIsTablePresent(Connection conn, String tablesTable, String schemaColumn, String tableNameColumn, String schemaName, String tableName) throws JdbcException {
-		String query="select count(*) from "+tablesTable+" where upper("+tableNameColumn+")=?";
+	public boolean doIsTablePresent(Connection conn, String tablesTable, String schemaColumn, String tableNameColumn, String schemaName, String tableName) throws JdbcException {
+		boolean isPresent = Boolean.FALSE;
 		if (StringUtils.isNotEmpty(schemaName)) {
 			if (StringUtils.isNotEmpty(schemaColumn)) {
-				query+=" and upper("+schemaColumn+")='"+schemaName.toUpperCase()+"'";
+				isPresent = this.isTablePresent(conn, tablesTable, schemaColumn);
 			} else {
 				throw new JdbcException("no schemaColumn present in table ["+tablesTable+"] to test for presence of table ["+tableName+"] in schema ["+schemaName+"]");
 			}
 		}
 		try {
-			return JdbcUtil.executeIntQuery(conn, query, tableName.toUpperCase())>=1;
+			isPresent = this.isTablePresent(conn, tablesTable);
 		} catch (Exception e) {
 			log.warn("could not determine presence of table ["+tableName+"]",e);
-			return false;
 		}
+		return isPresent;
 	}
 
-	private final boolean useMetaDataForTableExists=false;
+	
 	@Override
 	public boolean isTablePresent(Connection conn, String tableName) throws JdbcException {
-		try {
-			PreparedStatement stmt = null;
-			if (useMetaDataForTableExists) {
-				DatabaseMetaData dbmeta = conn.getMetaData();
-				ResultSet tableset = dbmeta.getTables(null, null, tableName, null);
-				return !tableset.isAfterLast();
-			} 
-			String query=null;
+	    return isTablePresent(conn, tableName, null);
+	}
+	
+	private boolean isTablePresent(Connection conn, String tableName, String schemaName) throws JdbcException {
+		boolean tExists = Boolean.FALSE;
+		ResultSet rs = null;
+	    try {
+	    	rs = conn.getMetaData().getTables(null, schemaName, tableName, null);
+	    	tExists = rs.next();
```

Comment:
Please have a look at the original code. You should use isAfterLast() instead of next()